src/Nri/Ui/Radio/V1.elm

```diff
@@ -0,0 +1,228 @@
+module Nri.Ui.Radio.V3
```

Comment:
YES! 😻 

Should this be a `V1`, considering its the first version of this module? 

**EDIT**: I see this change got undone in a next commit. Out of curiosity, was that out of concerns for this PR to grow to big or something else?

Reply:
I was wrong about this being meant to represent a radio button, actually! The "Round" style was used to represent checkmarks on the topic sentences outlines in the Rate step, and that seemed like a really specialized use that was overcomplicating the checkmark. 

Also, in the monolith there's already a RadioButton component -- seems like that should be added to noredink-ui separately?

Reply:
Ah, that makes sense, thanks for explaining!

Concerning these styles representing one-off implementations for various parts of the codebase, like the round style, happy to see them removed, it simplified things a lot! Do you have some ideas how widgets currently using these one-of themes in the monolith should be implemented?